### ISSUE TITLE
Why only unix systems?

### ISSUE THREAD

**Issue by kptlronyttcna** [rouzwawi/grpc-kotlin#29]
Is there something really hard about supporting windows? If you know what's the issue, maybe I can help.

**Comment by rouzwawi** [comment]
@kptlronyttcna It's just that I don't have access to a Windows environment to test it. I believe you need to attach additional artifacts to the `-gen` module that are executable on windows. It should be pretty similar to how they are adressed for the grpc generator (`${os.detected.classifier}` part):

```
io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}
```

Seems like they are producing these artifacts with gradle: https://github.com/grpc/grpc-java/blob/master/compiler/build.gradle#L233-L240

**Comment by fatfatson** [comment]
@rouzwawi 
still don't know where to modify...could you give more hints? I'd like to build my project with this plugin in windows.

**Comment by fatfatson** [comment]
I have managed to build the exe by `launch4j`, then set its path to the grpckotlin plugin config in gradle
,
so I could build my project in windows
but I still don't know how to publish it..

**Comment by rouzwawi** [comment]
This was fixed with #44 and is available from `v0.1.3`. See instructions in README for how to run the plugin.
